test/games/strategy/triplea/ui/TestRoute.java

```diff
@@ -0,0 +1,64 @@
+package games.strategy.triplea.ui;
```

Comment:
Part of the problem is dependency injection, being able to insert mock objects as appropriate. Static is bad for that, creates some tight coupling.

Here is one thing you can do:
```
private static MyLogicHelperClass helperClass = new DefaultInstance();
protected static void setHelperClass(MyLogicHelperClass mockValue) {
   helperClass = mockValue;
}
```
There are still some problems there, but that would let you break up the static logic across multiple classes. Do avoid using powermockito to mock statics, generally there are easier/better ways to achieve that goal. Push comes to shove, powermockito is an option, but should be last resort.